### ISSUE TITLE
Player strip menu title doesnt update on identity change

### ISSUE THREAD

**Issue by lzk228** [space-wizards/space-station-14#29446]

**Comment by Aexxie** [comment]
This one needs more context. Do you mean it doesn't update when you change your identity with an Agent ID?

*I also noticed a more prominent issue while looking into this one, probably related.*
Admin Command `rename` doesn't work on humanoids and the likes; when viewing as a ghost or admin, it shows the humanoid as if they had their face covered and have someone else's ID.

**Non-Admin/Player view:**
![image](https://github.com/space-wizards/space-station-14/assets/125095677/782cbdce-8c28-4dbd-82c1-b9a39bb6cca0)
![image](https://github.com/space-wizards/space-station-14/assets/125095677/fd18b1cf-8939-4fbe-8135-e1c4a48309df)

**Admin/Ghost view:**
![image](https://github.com/space-wizards/space-station-14/assets/125095677/f3aa04c4-3ca6-455b-a6f0-6a3bd16063a4)
![image](https://github.com/space-wizards/space-station-14/assets/125095677/f7d83130-27a8-48c9-b215-bd7a20fe2541)

This also affects the "rename" trick.

**Comment by lzk228** [comment]
i mean strip menu, for example when you open Urist's strip menu, you see Urist McHands in title, but if they change identity (wear mask and Pun Pun id) the title is still Urist McHands

**Comment by lzk228** [comment]
here i provided a video

https://github.com/space-wizards/space-station-14/assets/124214523/d6f86f35-be79-4b5a-a5d4-eec065d614f7

**Comment by Tayrtahn** [comment]
Yeah, I noticed this with the name modifier stuff. The window title doesn't update when the target's name changes.

Recent engine work added an event that gets raised when an entity's name is changed. It might be as easy as subscribing to that on the client and updating the window, but I'm not familiar enough with UI stuff to be sure.
